### PR TITLE
Clean up 

### PR DESCRIPTION
**Pull request by kakwok**
Implements #125 and  #29

### CODE REVIEW

**Comment by kakwok** [comment]
Conflicts will be resolved when 18.1.3 is deployed to global.

File: src/rcms/fm/app/level1/HCALEventHandler.java [hunk 1]
```diff
@@ -545,7 +512,6 @@ protected void waitforTriggerAdapter(int TimeOut) {
               }
               catch (XDAQTimeoutException e) {
                 String errMessage = "[HCAL " + functionManager.FMname + "] Error! XDAQTimeoutException: waitforTriggerAdapter()\n Perhaps the trigger adapter application is dead!?";
-                logger.error(errMessage,e);
```
Comment (jhakala) [inline src/rcms/fm/app/level1/HCALEventHandler.java]: Why was this line removed?

**Comment by kakwok** [comment]
Forgot to pull integration again for an old branch when I assumed no conflicts.
Fast forwarded from integration in 795be68
